client/app/reducers/selection.coffee

```diff
@@ -0,0 +1,38 @@
+Immutable = require 'immutable'
+
+{ActionTypes} = require '../constants/app_constants'
+
+DEFAULTSTATE =
```

Comment:
I think that we should use underscore separated words in all our constants (`DEFAULT_STATE` instead of `DEFAULTSTATE`), like for example `MAILBOX_SELECT_ALL` below.